test/controllers/about_controller_test.exs

```diff
@@ -1,8 +1,31 @@
 defmodule ResearchResource.AboutControllerTest do
   use ResearchResource.ConnCase
 
+  setup %{conn: conn} = config do
+    if username = config[:login_as] do
+      user = insert_user(email: username)
+      conn = assign(build_conn(), :current_user, user)
+      {:ok, conn: conn, user: user}
+    else
+      :ok
+    end
+  end
+
```

Comment:
I've tried to extract the code inside the setup in a function but I get some conflict when I want to use the assign function with ConCase outside the tests. I haven't find a solution yet and it doesn't seem as obvious as I was thinking. Do you have any idea how to do this @shouston3 ?